templates/Publisher.cpp.em

```diff
@@ -108,6 +108,8 @@ bool @(topic)_Publisher::init(const std::string& ns)
 
     // Create Publisher
     PublisherAttributes Wparam;
+    Wparam.qos.m_durability.durabilityKind(VOLATILE);
+    Wparam.qos.m_reliability.kind = eprosima::fastdds::dds::BEST_EFFORT_RELIABILITY_QOS;
```

Comment:
The namespace is already included in Publisher.h.em, (using namespace eprosima::fastdds::dds;) so probably it is not entirely correct to specify the full namespace here.

In older version of the fastrtps this qos setting is in namespace eprosima::fastrtps::rtps